integration/fuse/src/main/java/alluxio/fuse/cli/FuseShell.java

```diff
@@ -42,6 +47,24 @@
   public FuseShell(FileSystem fs, AlluxioConfiguration conf) {
     mFileSystem = fs;
     mConf = conf;
+    mCommands = loadCommands();
```

Comment:
Is it possible to let `FuseShell` extends from `AbstractShell`?

Reply:
The two class's run method return value is not compatible. Our return value is a mocked 'URIStatus' type , while AbstractShell return is 'int' type.